code/modules/recycling/sortingmachinery.dm

```diff
@@ -458,9 +458,9 @@
 	. = ..()
 	var/potential_cut = input("How much would you like to payout to the registered card?","Percentage Profit") as num|null
 	if(!potential_cut)
-		percent_cut = 50
-	percent_cut = clamp(round(potential_cut, 1), 1, 50)
-	to_chat(user, "<span class='notice'>[percent_cut]% profit will be received if a package with a barcode is sold.</span>")
+		cut_multiplier = 0.2
+	cut_multiplier = clamp(round(potential_cut, 1), 1, 50) / 100
+	to_chat(user, "<span class='notice'>[cut_multiplier*100]% profit will be received if a package with a barcode is sold.</span>")
```

Comment:
I foresee weird things with floats should the number ever be something not pretty (10.0000000000000000000001% profit). I recommend rounding this number before you display it.